src/app/src/main/kotlin/org/vaccineimpact/api/app/security/JWTParameterClient.kt

```diff
@@ -0,0 +1,28 @@
+package org.vaccineimpact.api.app.security
+
+import org.pac4j.http.client.direct.ParameterClient
+import org.vaccineimpact.api.app.repositories.RepositoryFactory
+import org.vaccineimpact.api.models.ErrorInfo
+import org.vaccineimpact.api.security.OneTimeTokenAuthenticator
+import org.vaccineimpact.api.security.OneTimeTokenChecker
+import org.vaccineimpact.api.security.WebTokenHelper
+
+// This client receives the token as TokenCredentials and stores the result as JwtProfile
+class JWTParameterClient(helper: WebTokenHelper, oneTimeTokenChecker: OneTimeTokenChecker)
+    : ParameterClient("access_token", OneTimeTokenAuthenticator(helper, oneTimeTokenChecker))
+{
+    init
+    {
+        @Suppress("UsePropertyAccessSyntax")
+        this.setSupportGetRequest(true)
```

Comment:
Why not use property access syntax?

Reply:
Because then it reads `isSupportGetRequest = true`, which feels less readable to me here. The `is` prefix makes sense when it's something like `getEnabled`/`setEnabled` and it changes to `isEnabled`. But not so much here.

Reply:
I thought this was why. That's fine